game-app/game-core/src/main/resources/i18n/games/strategy/engine/framework/ui.properties

```diff
@@ -0,0 +1,23 @@
+startup.SetupPanelModel.button.ConnectToNetworkedGame.Label=Connect to Networked Game
+startup.SetupPanelModel.button.EnginePreferences.Label=Engine Preferences
+startup.SetupPanelModel.button.EnginePreferences.Tooltip=Configure certain options related to the engine.
+startup.SetupPanelModel.button.HostNetworkGame.Label=Host Networked Game
+startup.SetupPanelModel.button.MapCreatorTools.Label=Map Creator Tools
+startup.SetupPanelModel.button.PlayByEmail.Label=Play By Email
+startup.SetupPanelModel.button.PlayByEmail.Tooltip=Starts a game which will be emailed back and forth between all players.
+startup.SetupPanelModel.button.PlayByForum.Label=Play By Forum
+startup.SetupPanelModel.button.PlayByForum.Tooltip=Starts a game which will be posted to an online forum or a message board.
+startup.SetupPanelModel.button.PlayOnline.ConnectToNetworkedGame.Tooltip.Line1=Connects to someone's hosted game,
+startup.SetupPanelModel.button.PlayOnline.ConnectToNetworkedGame.Tooltip.Line2=so long as you know their IP address.
+startup.SetupPanelModel.button.PlayOnline.HostNetworkGame.Tooltip.Line1=Hosts a network game, which people can connect to.
+startup.SetupPanelModel.button.PlayOnline.HostNetworkGame.Tooltip.Line2=Anyone on a LAN will be able to connect.
+startup.SetupPanelModel.button.PlayOnline.HostNetworkGame.Tooltip.Line3=Anyone from the internet can connect as well, but only if the host has configured port forwarding correctly. 
+startup.SetupPanelModel.button.PlayOnline.Label=Play Online
+startup.SetupPanelModel.button.PlayOnline.Tooltip.Line1=Find Games Online on the Lobby Server.
+startup.SetupPanelModel.button.PlayOnline.Tooltip.Line2=TripleA is MEANT to be played Online against other humans.
+startup.SetupPanelModel.button.PlayOnline.Tooltip.Line3=Any other way is not as fun!
+startup.SetupPanelModel.button.StartLocalGame.Label=Start Local Game
+startup.SetupPanelModel.button.StartLocalGame.Tooltip.Line1=Start a game on this computer.
+startup.SetupPanelModel.button.StartLocalGame.Tooltip.Line2=You can play against a friend sitting besides you (hotseat mode),
+startup.SetupPanelModel.button.StartLocalGame.Tooltip.Line3=or against one of the AIs.
```

Comment:
I really dislike this separation of lines, it makes it really hard to use it in languages that aren't too similar too english and might have huge differences in sentence length.
Instead I'd suggest making use of `\n` in the `.properties` File and replacing those if `<br>` is required instead. This way the whole system is less dependant on certain language features overall.

I'd recommend having a look at https://projectfluent.org/ they provide some examples of why translating software can be challenging (also IIRC this is the system used in Firefox and other Mozilla Software)

Reply:
Agree, the separation of lines should be in the properties file.
To avoid replace logic, which is not always necessary, I put the <br/> tag in the properties file.